src/pages/Login.tsx

```diff
@@ -0,0 +1,41 @@
+import './Login.css'
+
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom';
+
+import { Button } from "@/components/common/Button"
+import { UnderlineTextField } from "@/components/common/Form/Input/UnderlineTextField"
+import { useAuth } from '@/context/AuthContext';
+
+const Login = () => {
+
+    const navigate = useNavigate()
+    const { login } = useAuth()
+    const [id, setId] = useState('')
+    const [pw, setPw] = useState('')
+
+    const handleLogin = () => {
+        if (id&&pw) { // id와 비밀번호가 모두 입력되었을 때만 로그인 처리
+            login(id)
+            navigate(-1)
+        } else {
+            alert('ID와 비밀번호를 모두 입력해주세요.')
+        }
+
+    }
+
+  return (
+    <div className="Login">
+        <div className="title">Kakao</div>
+        <div className="container">
+            <UnderlineTextField placeholder='이름' value={id} onChange={(e) => setId((e.target as HTMLInputElement).value)} className='textFieldId'/>
```

Comment:
```suggestion
            <UnderlineTextField placeholder='이름' value={id} onChange={(e) => setId(e.target.value)} className='textFieldId'/>
```

[UnderlineTextField](https://github.com/yunn23/react-gift-ui-flow/blob/2bc7bdca9690d2af20367230939585e9184cf1ba/src/components/common/Form/Input/UnderlineTextField.tsx#L9)에서 이미 `input`으로 식별되도록 해뒀기 때문에 `as`로 적용한 [type assertion](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions)은 제거해도 되겠습니다. 꼭 필요한 경우가 아니라면 사용하지 않는걸 추천드려요.